insights/parsers/vdo_status.py

```diff
@@ -0,0 +1,209 @@
+                bios acknowledged partial fua: 0
+                bios acknowledged partial read: 0
+                bios acknowledged partial write: 0
+                slab journal tail busy count: 0
+                slab summary blocks written: 0
+                slabs opened: 0
+                slabs reopened: 0
+                updates found: 0
+                updates not found: 0
+                logical blocks: 1046277
+                logical blocks used: 0
+                maximum VDO IO requests in progress: 9
+                maximum dedupe queries: 0
+                no space error count: 0
+                operating mode: normal
+                overhead blocks used: 787140
+                physical blocks: 1835008
+                data blocks used: 0
+                ...
+          vdo2:
+            Acknowledgement threads: 1
+            Activate: enabled
+            ...
+            VDO statistics:
+              /dev/mapper/vdo1:
+                1K-blocks: 7340032
+                ...
+
+    Examples:
+        >>> type(vdo)
+        <class 'insights.parsers.vdo_status.VDOStatus'>
+        >>> vdo['Kernel module']['Name']
+        'kvdo'
+        >>> vdo['Configuration']['File']
+        '/etc/vdoconf.yml'
+        >>> vdo['VDOs']['vdo1']['Activate']
+        'enabled'
+        >>> vdo['VDOs']['vdo1']['VDO statistics']['/dev/mapper/vdo1']['1K-blocks']
+        7340032
+        >>> vdo['VDO status']
+        {'Date': '2019-07-24 20:48:16-04:00', 'Node': 'dell-m620-10.rhts.gsslab.pek2.redhat.com'}
+        >>> vdo['VDOs']['vdo2']['Acknowledgement threads']
+        1
+        >>> vdo.get_slab_size('vdo1')
+        '2G'
+        >>> vdo.volumns
+        ['vdo1', 'vdo2']
+        >>> vdo.get_physical_blocks('vdo1')
+        1835008
+        >>> vdo.get_physical_used('vdo1')
+        0
+        >>> vdo.get_physical_free('vdo1')
+        1047868
+        >>> vdo.get_logical_used('vdo1')
+        0
+        >>> vdo.get_overhead_used('vdo1')
+        787140
+
+    Attributes:
+
+        data (dict): the result parsed of 'vdo status'
+
+        volumns (list): The list the vdo volumns involved
+
+        dmapper (dict): The dict of device mapper vdo under 'VDO statistics'
+
+    Raises:
+        ParseException: When input content is not available to parse
+
+    """
+
+    def __init__(self, content):
+        YAMLParser.__init__(self, content)
+        self.volumns = []
+        try:
+            keys = list(self.data['VDOs'].keys())
+            keys.sort()
+            self.volumns = keys
+        except:
+            raise ParseException("couldn't parse yaml")
+
+    def __get_dev_mapper__(self, vol):
+        try:
+            dm_path = ('/dev/mapper/%s' % vol)
+            return self.data['VDOs'][vol]['VDO statistics'][dm_path]
+        except:
+            raise ParseException("couldn't parse yaml")
+
+    def get_all_volumns(self):
+        """list: vdo volumns"""
+        return self.volumns
```

Comment:
`return sorted(self.data['VDOs'].keys()) if 'VDOs' in self.data else []`

If OK to you, you could also change the method to a property which has the same usage to attribute.

```python
@property
def volumns(self)
  return sorted(self.data['VDOs'].keys()) if 'VDOs' in self.data else []
```